src/App.tsx

```diff
@@ -15,142 +85,41 @@ export const App: React.FC = () => {
 
       <div className="todoapp__content">
         <header className="todoapp__header">
-          {/* this buttons is active only if there are some active todos */}
-          <button type="button" className="todoapp__toggle-all active" />
+          {isAnyActiveTodo && (
+            <button type="button" className="todoapp__toggle-all active" />
```

Comment:
Add an onClick event handler to the 'todoapp__toggle-all' button to mark all todos as completed or active.